gradle.properties

```diff
@@ -4,8 +4,8 @@ version=2.5.2-SNAPSHOT
 ballerinaLangVersion=2201.3.0
 ballerinaTomlParserVersion=1.2.2
 commonsLang3Version=3.8.1
-nettyVersion=4.1.77.Final
-nettyTcnativeVersion=2.0.52.Final
+nettyVersion=4.1.86.Final
+nettyTcnativeVersion=2.0.54.Final
```

Comment:
We need to update the netty-tcnative versions in the `Ballerina.toml` right?

Reply:
Fixed via cf88f30b8d33b1b246b6f32ba01fdeecbff48d59